README.rst

```diff
@@ -240,6 +240,59 @@ inside the current folder. If you omit the final ``tmp/`` you'll leave the
 empty ``tmp`` directory in place. (**WARNING!** Don't put the ``.`` *after*
 the ``--erase`` option! Obviously, your project files will all be deleted.)
 
+🏷️ Git-clean integration
+--------------------------
+
+PyClean integrates with Git's built-in ``git clean`` command to remove
+untracked files from your working directory. This provides a user-friendly
+way to automatically clean up files that are not under version control.
+
+**Requirements:** Git must be installed on your system for this feature to
+be available.
+
+The ``--git-clean`` (or ``-g``) flag runs ``git clean`` after all other
+cleanup operations (bytecode, debris, and free-form targets). The flag
+honors PyClean's ``--ignore``, ``--dry-run``, and ``--yes`` options:
+
+- **Interactive mode (default):** Git prompts you to confirm which files
+  to delete.
+- **Dry-run mode (--dry-run):** Shows which files would be deleted without
+  actually removing them.
+- **Force mode (--yes):** Deletes all untracked files without prompting.
```

Comment:
This is too verbose and partially an implementation detail. Make it more concise.